examples/nlp/duplex_text_normalization/google_data_preprocessing.py

```diff
@@ -15,7 +15,7 @@
 """
 This script can be used to process the raw data files of the Google Text Normalization dataset
 to obtain data files of the format mentioned in the `text_normalization doc <https://github.com/NVIDIA/NeMo/blob/main/docs/source/nlp/text_normalization.rst>`.
```

Comment:
could you add this script supports EN, and RU Google Text norm dataset processing?

Reply:
did you had to do some german processing? if not im ok to merge PR

Reply:
No. I directly used your German's dataset. The original Google dataset also does not have German data.